spec/autoload.php

```diff
@@ -8,7 +8,7 @@
     ]);
 }
 
-if (PHP_VERSION_ID >= 70300 && PHP_VERSION_ID < 70400) {
+if (PHP_VERSION_ID >= 70300) {
```

Comment:
this was a bit odd. @Nyholm would you remember if this was done for a specific reason?

if i understand correctly, this is only for the tests, so we could rename the classes without isuses. if the semantic of the if is correct, AtLeastPHP73 and LessThanPHP73 would be better names.

if we would ever need an upper bound, we can introduce it and rename the classes again.

Reply:
At the time Puli didn;t work with PHP 7.3 yet, and that was the latest version. 2 years later, we are now in the siutation where 7.4 and 8.0 don't work either. I guess I can rename this class if we want. Let me know what name you prefer. :)

Reply:
As for what this class does, yes, it exists so we can skip the puli tests.

Reply:
yep. lets call them AtLeastPHP73 and LessThanPHP73? or possibly: PuliCompatible and PuliIncompatible (if we even need both and not only one)

Reply:
Already renamed it around an hour ago btw. :)